src/ts/components/settingspanel.ts

```diff
@@ -152,11 +294,21 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
 
   /**
    * Gets the event that is fired when one or more {@link SettingsPanelItem items} have changed state.
+   * @deprecated
```

Comment:
Why is this deprecated? Can't it be removed? A deprecated element in a major release does not make a lot of sense.